koans/AboutApplyingWhatWeHaveLearnt.js

```diff
@@ -83,7 +87,9 @@ describe("About Applying What We Have Learnt", function() {
 
```

Comment:
On the line before this where it says `var ingredientCount = { "{ingredient name}": 0 };` you should use a mix of `flatten` and `reduce`.